### PR TITLE
feat: added clone service feature

### PR DESCRIPTION
**Pull request by acarranoqovery**
added clone feature service. 

note it has been added as template instead of copying the text around the different services

### CODE REVIEW

File: scripts/generate/templates/_partials/_clone_service.md.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+You can create a clone of the service via the clone feature. A new service with the same configuration (see below for exceptions) will be created into the target environment.
+
+<p align="center">
+  <img src="/img/clone_service.png" alt="Clone Service" />
+</p>
+
+The target environment can be the same as the current environment or even another one in a completely different project.
+
+** Importan information **
```
Comment (ctjhoa) [inline scripts/generate/templates/_partials/_clone_service.md.erb:9]:
```suggestion
** Important information **
```

File: website/docs/using-qovery/configuration/application.md [hunk 2]
```diff
@@ -447,6 +447,29 @@ To learn how to display your application logs, navigate to [logs section][docs.u
...
+
+You can create a clone of the service via the clone feature. A new service with the same configuration (see below for exceptions) will be created into the target environment.
+
+<p align="center">
+  <img src="/img/clone_service.png" alt="Clone Service" />
+</p>
+
+The target environment can be the same as the current environment or even another one in a completely different project.
+
+** Importan information **
```
Comment (ctjhoa) [inline website/docs/using-qovery/configuration/application.md:460]:
```suggestion
** Important information **
```

File: website/docs/using-qovery/configuration/cronjob.md [hunk 3]
```diff
@@ -287,6 +287,28 @@ To learn how to set up secrets in your projects and applications, navigate to [c
...
+
+You can create a clone of the service via the clone feature. A new service with the same configuration (see below for exceptions) will be created into the target environment.
+
+<p align="center">
+  <img src="/img/clone_service.png" alt="Clone Service" />
+</p>
+
+The target environment can be the same as the current environment or even another one in a completely different project.
+
+** Importan information **
```
Comment (ctjhoa) [inline website/docs/using-qovery/configuration/cronjob.md:300]:
```suggestion
** Important information **
```

File: website/docs/using-qovery/configuration/database.md [hunk 4]
```diff
@@ -225,6 +225,35 @@ Please note that the built-in variables follow the naming pattern: `QOVERY_DATAB
...
+
+You can create a clone of the service via the clone feature. A new service with the same configuration (see below for exceptions) will be created into the target environment.
+
+<p align="center">
+  <img src="/img/clone_service.png" alt="Clone Service" />
+</p>
+
+The target environment can be the same as the current environment or even another one in a completely different project.
+
+** Importan information **
```
Comment (ctjhoa) [inline website/docs/using-qovery/configuration/database.md:238]:
```suggestion
** Important information **
```

File: website/docs/using-qovery/configuration/lifecycle-job.md [hunk 5]
```diff
@@ -393,6 +393,29 @@ Var `QOVERY_OUTPUT_JOB_<JOBID>_DB_PORT`
...
+
+You can create a clone of the service via the clone feature. A new service with the same configuration (see below for exceptions) will be created into the target environment.
+
+<p align="center">
+  <img src="/img/clone_service.png" alt="Clone Service" />
+</p>
+
+The target environment can be the same as the current environment or even another one in a completely different project.
+
+** Importan information **
```
Comment (ctjhoa) [inline website/docs/using-qovery/configuration/lifecycle-job.md:406]:
```suggestion
** Important information **
```

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: scripts/generate/templates/_partials/_clone_service.md.erb (hunk 1 above)
Comment (acarranoqovery) [reply scripts/generate/templates/_partials/_clone_service.md.erb]: fixed

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**